dtm0/dtx.h

```diff
@@ -79,18 +89,25 @@ M0_INTERNAL void m0_dtx0_free(struct m0_dtx *dtx);
 M0_INTERNAL int m0_dtx0_prepare(struct m0_dtx *dtx);
 
 M0_INTERNAL int m0_dtx0_open(struct m0_dtx  *dtx, uint32_t nr);
-M0_INTERNAL int m0_dtx0_assign(struct m0_dtx       *dtx,
-			       uint32_t             pa_idx,
-			       const struct m0_fid *pa_fid);
+M0_INTERNAL int m0_dtx0_assign_fid(struct m0_dtx       *dtx,
+				   uint32_t             pa_idx,
+				   const struct m0_fid *pa_fid);
+M0_INTERNAL void m0_dtx0_assign_fop(struct m0_dtx       *dtx,
+				    uint32_t             pa_idx,
+				    const struct m0_fop *pa_fop);
 M0_INTERNAL int m0_dtx0_close(struct m0_dtx *dtx);
 
+M0_INTERNAL void m0_dtx0_executed(struct m0_dtx *dtx, uint32_t pa_idx);
+
 /* Puts a copy of dtx's transaction descriptor into "dst".
```

Comment:
[* question *] Have we moved away from doxygen comments? must be  `/**` for a doxygen comments, no?